.github/workflows/build_documentation.yml

```
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr zh-CN zh-TW
+      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr zh-CN zh-TW vi
```

Could we please order these alphabetically?

```suggestion
      languages: ar bn de en es fa fr gj he hi it ja ko pt ru th tr vi zh-CN zh-TW
```